frontend/src/pages/Player/Toolbar/DevToolsWindowV2/NetworkPage/NetworkPage.tsx

```diff
@@ -419,7 +419,6 @@ const ResourceRow = ({
 						className={styles.timingBar}
 						style={{
 							width: `${actualPercent}%`,
-							zIndex: 100,
```

Comment:
To avoid this:
<img width="306" alt="Screenshot 2023-02-07 at 9 08 05 AM" src="https://user-images.githubusercontent.com/17913919/217314441-faf1a6b0-cd62-4398-88d1-c3eed2f612cd.png">